tests/src/test/java/org/wso2/micro/gateway/tests/common/CLIExecutor.java

```diff
@@ -243,6 +243,6 @@ public static CLIExecutor getInstance() {
      */
     public String getLabelBalx(String project) {
```

Comment:
Should we rename the method?
```suggestion
    public String getLabelJar(String project) {
```

Reply:
Fixed with PR : https://github.com/wso2/product-microgateway/pull/791